indico/modules/events/management/templates/delete_event.html

```diff
@@ -1,10 +1,18 @@
 {% from 'confirmation_dialog.html' import confirmation_dialog %}
 
 {% set confirmation_message %}
-    {% trans -%}
-        Please note that if you delete the event you will lose all the information contained in it.
-        This operation is irreversible!
-    {%- endtrans %}
+    {% if event.reservations %}
```

Comment:
`{% set num_bookings = event.all_room_reservation_links.count() %}` and use this here and below